tao_compiler/mlir/xla/ral/BUILD

```diff
@@ -161,15 +162,31 @@ cc_library(
     deps = if_platform_alibaba([":real_disc_patine_client_deps"])
 )
 
+cc_library(
+    name = "blade_gemm_deps",
+    deps = if_torch_disc(
+        [
+            "@blade_gemm//:blade_gemm"
+        ],
+        [
+            "//tao/blade_gemm:blade_gemm"
```

Comment:
As offline discussed, the tf tao_compiler target has no dependency on this target. If the torch ral_context target goes to the if_true branch of if_torch_disc, maybe this if_false branch as well as the build_blade_gemm() in tao_build.py can be removed?

Reply:
so would the cmake branch and the cmake building code in `common_setup.py` get removed?

Reply:
Removed in latest commit for this pr

Reply:
While we do not need this for `tao_compiler_main` build, `disc-replay-main` still need this, still need some refactor for `disc-replay-main` to be able to build.